templates/package.json

```diff
@@ -1,53 +1,52 @@
 {
-  "name": "@logto/connector-apple",
-  "version": "1.0.0-beta.12",
-  "description": "Apple web connector implementation.",
   "main": "./lib/index.js",
   "exports": "./lib/index.js",
-  "author": "Silverhand Inc. <contact@silverhand.io>",
   "license": "MPL-2.0",
   "files": [
     "lib",
     "docs",
     "logo.svg",
-    "README.md",
```

Comment:
Why are we removing README.md here?

Reply:
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files